incubator/ethereum/templates/bootnode.deployment.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: apps/v1beta2 
+kind: Deployment
+metadata:
+  name: {{ template "ethereum.fullname" . }}-bootnode-deployment
+  labels:
+    app: {{ template "ethereum.name" . }}-bootnode-deployment
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ template "ethereum.name" . }}-bootnode
```

Comment:
The `app` label should be `{{ template "ethereum.name" . }}` on all resources the chart creates. You should add `component` labels that should match the suffix of the resource names. Label selectors should then have `app`, `release`, and `component` labels.

See https://github.com/unguiculus/charts/blob/master/REVIEW_GUIDELINES.md for details.

Reply:
Fixed.